core/state_transition.go

```diff
@@ -412,6 +413,10 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 		ret, st.gas, vmerr = st.evm.Call(sender, st.to(), st.data, st.gas, st.value)
 		stateTransitionEvmCallExecutionTimer.Update(time.Since(evmCallStart))
 	}
+	var errL1 *vm.ErrL1RPCError
+	if errors.As(vmerr, &errL1) {
+		return nil, vmerr
+	}
```

Comment:
Have you tested how this will work? Will the node reject the block or reexecute it?